src/features/hotspots/details/HotspotDetailCardHeader.tsx

```diff
@@ -1,36 +1,77 @@
 import React from 'react'
+import { Linking, Share } from 'react-native'
+import { useSelector } from 'react-redux'
+import { useActionSheet } from '@expo/react-native-action-sheet'
+import { useTranslation } from 'react-i18next'
 import Box from '../../../components/Box'
 import CardHandle from '../../../components/CardHandle'
-import Heart from '../../../assets/images/heart.svg'
+// import Heart from '../../../assets/images/heart.svg'
 import MoreMenu from '../../../assets/images/moreMenu.svg'
 import TouchableOpacityBox from '../../../components/TouchableOpacityBox'
+import { useAppDispatch } from '../../../store/store'
+import hotspotDetailsSlice from '../../../store/hotspotDetails/hotspotDetailsSlice'
+import { RootState } from '../../../store/rootReducer'
 
-type Props = {
-  onMoreSelected: () => void
-  onFollowSelected: () => void
-}
+const HotspotDetailCardHeader = () => {
+  const { t } = useTranslation()
+  const dispatch = useAppDispatch()
+  const { showActionSheetWithOptions } = useActionSheet()
+
+  const {
+    hotspotDetails: { hotspot },
+  } = useSelector((state: RootState) => state)
+
+  type SettingsOption = { label: string; action?: () => void }
+  const onMoreSelected = () => {
+    if (!hotspot) return
```

Comment:
This click isnt working for me, for some reason hotspot is always undefined

Reply:
hm I think it's because that card header doesn't get rerendered when the hotspot changes, probably for good reason. I think I'll need to move that down below the handle